doc/BUILDSERVER.md

```diff
@@ -0,0 +1,160 @@
+# Buildserver
+
+The [Elektra buildserver](https://build.libelektra.org/) handles a variety of
+tasks that reaches from testing PR's to deploying new versions of the Elektra
+homepage.
+
+## Legacy build jobs
+Build jobs that are not ported into Jenkinsfiles use
```

Comment:
Is build jobs the same as tasks?

Reply:
A build job is a Jenkins term. For example: libelektra is a job, elektra-gcc is a job.
Task has no special meaning and represents the english word i.e. 'a piece of work to be done or undertaken'.

Reply:
Yes, these infos should be in the docu. It must be clear what (Jenkins/Docker) terms are and which words are used in their normal meaning. Italics can help. For me it was unclear if there are tasks (done by the buildserver) that are not build jobs?